setup.cfg

```diff
@@ -18,4 +18,5 @@ pep8ignore =
     E226    # missing white space around arithmetic operator
     E222    # multiple spaces after operator
     W391    # blank line at the end of file
+    E701    # multiple statements on one line
```

Comment:
Why ?

Reply:
form some reason I can't deactivate it locally (not working) and I need this declaration on staggered orders:
```python
@dataclass
class OrderData:
    side: TradeOrderSide = None
    quantity: float = 0
    price: float = 0
    symbol: str = 0
    is_virtual: bool = True
```

Reply:
Did you try to add `# pylint: disable=E701` at the very top of the file ?